scripts/monitoring/pcp_sample.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python
```

Comment:
This should be put in openshift-tools/scripts/monitoring and packaged in an RPM.

The other sends need to be packaged as well, they were just quick and dirty, but we definitely don't want to add to the tech debt here.